HotSpot/View Controllers /ReportDriverViewController.m

```diff
@@ -0,0 +1,99 @@
+//
+//  ReportDriverViewController.m
+//  
+//
+//  Created by aaronm17 on 8/6/19.
+//
+
+#import "ReportDriverViewController.h"
+#import <Parse/Parse.h>
+#import "RegexHelper.h"
+#import "EmailHelper.h"
+#import "ImagePickerHelper.h"
+
+@interface ReportDriverViewController () <UIImagePickerControllerDelegate, UINavigationControllerDelegate>
+
+@property (weak, nonatomic) IBOutlet UIImageView *carImage;
+@property (weak, nonatomic) IBOutlet UITextField *licensePlate;
+@property (weak, nonatomic) IBOutlet UITextView *reportMessage;
+
+@end
+
+@implementation ReportDriverViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (IBAction)didTapCancel:(UIBarButtonItem *)sender {
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+//checks if the license plate is invalid, if not retrieves the driver
+//of that car and sends a specialized report
+- (IBAction)didTapSendReport:(UIButton *)sender {
+    UIAlertController *alert = [RegexHelper createAlertController];
+    if ([RegexHelper isEmpty:@"licensePlate"]) {
+        alert.title = @"Empty license plate";
```

Comment:
you can have a helper method that presents an alert given the title and message as arguments. would save you so many lines of code